docs/sources/datasources/prometheus/configure-prometheus-data-source.md

```diff
@@ -121,12 +123,14 @@ Following are additional configuration options.
 
 ### Performance
 
-- **Prometheus type** - The type of your Prometheus server. There are four options: `Prometheus`, `Cortex`, `Thanos`, `Mimir`.
+- **Prometheus type** - The type of your Prometheus server. There are four options: `Prometheus`, `Cortex`, `Mimir`, `Thanos`.
```

Comment:
I think these should be in bold if they are a UI element perhaps without any formatting: https://grafana.com/docs/writers-toolkit/write/style-guide/style-conventions/#bold

```suggestion
- **Prometheus type** - The type of your Prometheus server. There are four options: **Prometheus**, **Cortex**, **Mimir**, and **Thanos**.
```

Reply:
I think they're more of a configuration option than a direct UI element, but can you double check that?
https://grafana.com/docs/writers-toolkit/write/style-guide/style-conventions/#code

Throughout this documentation, it seems like those types of words have code formatting(`) in them.